ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1021,6 +1021,13 @@
     "UserRole": "caseworker-publiclaw-systemupdate",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
could we add this to non prod definition?

Reply:
good point, will do